scripts/ci/deploy-travis.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+###
+# Copyright 2017 resin.io
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+set -e
+set -u
+
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+  ./scripts/build/docker/run-command.sh -r $TARGET_ARCH -s $PWD -c "make publish-aws-s3"
```

Comment:
This should probably use the same "$HERE/...etc..." that other scripts use?

(Hmmm, would it be worth submitting a separate PR to update _all_ our scripts to use the "$HERE/...etc..." style of referencing paths?)